notebooks/NIRCAM/Imaging/requirements.txt

```diff
@@ -2,4 +2,5 @@ numpy<2.0
 jwst==1.15.1
 astroquery
 jdaviz
-astropy>=4.3.1
\ No newline at end of file
+astropy>=4.3.1
```

Comment:
why is the astropy requirement only added for the nircam and niriss notebooks?

Reply:
Not sure; I'd like to see if we can clean out some unnecessary requirements when we make the update to 1.17.1 in the next couple of weeks.  And yes, I tested that 1.15.1 installed ok with this updated requirements file, and that the gwcs version was appropriate :)

Reply:
Looks like these should be completely unnecessary, since jwst already pins astropy>5.3
Can iterate with the authors to remove as part of the 1.17.1 update.